app/config.py

```diff
@@ -87,6 +87,9 @@ class Config(object):
             "tts-benefits-studio@gsa.gov",
         ],
     }
+    FEATURE_BEST_PRACTICES_ENABLED = (
+        getenv("FEATURE_BEST_PRACTICES_ENABLED", "false") == "true"
```

Comment:
Probably doesn't matter, but this isn't standard python capitalization for bools